doc/source/whatsnew/v0.18.1.txt

```diff
@@ -66,6 +66,13 @@ Other Enhancements
    idx.take([2, -1])     # default, allow_fill=True, fill_value=None
    idx.take([2, -1], fill_value=True)
 
+- ``Index`` now supports ``.str.get_dummies()`` which returns ``MultiIndex`` (:issue:`10008`, :issue:`10103`)
+
```

Comment:
can you add this same example to the http://pandas.pydata.org/pandas-docs/stable/text.html#creating-indicator-variables ?